predicators/refinement_estimators/oracle_refinement_estimator.py

```diff
@@ -15,8 +15,13 @@ class OracleRefinementEstimator(BaseRefinementEstimator):
     def get_name(cls) -> str:
         return "oracle"
 
-    def get_cost(self, skeleton: List[_GroundNSRT],
+    @property
+    def is_learning_based(self) -> bool:
+        return False
+
+    def get_cost(self, initial_state: State, skeleton: List[_GroundNSRT],
                  atoms_sequence: List[Set[GroundAtom]]) -> float:
+        del initial_state  # unused
```

Comment:
I could be wrong, but I think that that this is unnecessary because it's overriding a parent class method -- I think that linting should be smart enough to realize that other subclasses might use that argument